File: src/Logger/LoggerSeverity.cpp
#include "./Include/LoggerSeverity.h"
namespace doc
{
    const LoggerSeverity LoggerSeverity::Trace = LoggerSeverity{0};
    const LoggerSeverity LoggerSeverity::Debug = LoggerSeverity{1};
    const LoggerSeverity LoggerSeverity::Info = LoggerSeverity{2};
    const LoggerSeverity LoggerSeverity::Warning = LoggerSeverity{3};
    const LoggerSeverity LoggerSeverity::Error = LoggerSeverity{4};
    const LoggerSeverity LoggerSeverity::Critical = LoggerSeverity{5};
    const LoggerSeverity LoggerSeverity::None = LoggerSeverity{6};
} 

File: go/docLogger/LoggerSeverity.go
package docLogger

type LoggerSeverity struct {
	value int
}

var (
	Trace    = LoggerSeverity{value: 0}
	Debug    = LoggerSeverity{value: 1}
	Info     = LoggerSeverity{value: 2}
	Warning  = LoggerSeverity{value: 3}
	Error    = LoggerSeverity{value: 4}
	Critical = LoggerSeverity{value: 5}
	None     = LoggerSeverity{value: 6}
)
